Diagnostic/Utils/XmlUtil.py

```diff
@@ -40,3 +42,10 @@ def addElement(xml,path,el,selector=[],addOnlyOnce=False):
 def createElement(schema):
     return ET.fromstring(schema)
 
+
+def removeElement(tree, parent_path, removed_element_name):
+    parents = tree.findall(parent_path)
+    for parent in parents:
+        removed_elements = parent.findall(removed_element_name)
+        for element in removed_elements:
+            parent.remove(element)
```

Comment:
I'm not sure if the two are different, but your suggested pattern seems correct too. I'll try that. After all, our use case has no multiple elements to be removed in the first place.